frontend/components/Form.js

```diff
@@ -1,11 +1,24 @@
 import React from 'react'
 
+//this allows the user to add to the todo + hide completed items or show all of the todos
 export default class Form extends React.Component {
   render() {
     return (
+      <form onSubmit={this.props.handleFormSubmit}>
       <div>
-        Form
+        <input 
+        type = 'text'
+        onChange={this.props.handleFormChanges} 
+        placeholder='Todo'/>
```

Comment:
Did you need a value here for the form? Where it's related to the props.input value in state